src/Actions/DispatchableFluxAction.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace FluxErp\Actions;
+
+use FluxErp\Contracts\ShouldBeMonitored;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+
+abstract class DispatchableFluxAction extends FluxAction implements ShouldBeMonitored, ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable;
+
+    public function handle(): mixed
+    {
+        return $this->withEvents()->validate()->execute();
```

Comment:
why does this even return anything? It is a job, not that the returned result is ever being used.

Reply:
possible to call it directly, if it returns void you cant embed this method into anything else where you need a return. this way its possible to call the handle without using it as a job